scripts/anvil_tdr_ingest/findability_subset/findability_subset_schema.json

```diff
@@ -0,0 +1,278 @@
+{
+  "tables": [
+    {
+      "name": "activity",
+      "columns": [
+        {"name": "activity_id", "datatype": "string", "array_of": false, "required": true},
+        {"name": "activity_type", "datatype": "string", "array_of": false},
+        {"name": "used_file_id", "datatype": "string", "array_of": true},
+        {"name": "generated_file_id", "datatype": "string", "array_of": true}
+      ],
+      "primaryKey": ["activity_id"],
+      "partitionMode": "none",
+      "datePartitionOptions": null,
+      "intPartitionOptions": null
+    },
+    {
+      "name": "alignmentactivity",
+      "columns": [
+        {"name": "alignmentactivity_id", "datatype": "string", "array_of": false, "required": true},
+        {"name": "activity_type", "datatype": "string", "array_of": false},
+        {"name": "data_modality", "datatype": "string", "array_of": true},
+        {"name": "generated_file_id", "datatype": "string", "array_of": true},
+        {"name": "used_file_id", "datatype": "string", "array_of": true},
+        {"name": "reference_assembly", "datatype": "string", "array_of": true}
+      ],
+      "primaryKey": ["alignmentactivity_id"],
+      "partitionMode": "none",
+      "datePartitionOptions": null,
+      "intPartitionOptions": null
+    },
+    {
+      "name": "antibody",
+      "columns": [
+        {"name": "antibody_id", "datatype": "string", "array_of": false, "required": true},
+        {"name": "target", "datatype": "string", "array_of": false}
+      ],
+      "primaryKey": ["antibody_id"],
+      "partitionMode": "none",
+      "datePartitionOptions": null,
+      "intPartitionOptions": null
+    },
+    {
+      "name": "assayactivity",
+      "columns": [
+        {"name": "assayactivity_id", "datatype": "string", "array_of": false, "required": true},
+        {"name": "activity_type", "datatype": "string", "array_of": false},
+        {"name": "antibody_id", "datatype": "string", "array_of": true},
+        {"name": "assay_type", "datatype": "string", "array_of": false},
+        {"name": "data_modality", "datatype": "string", "array_of": true},
+        {"name": "generated_file_id", "datatype": "string", "array_of": true},
+        {"name": "used_biosample_id", "datatype": "string", "array_of": true}
+      ],
+      "primaryKey": ["assayactivity_id"],
+      "partitionMode": "none",
+      "datePartitionOptions": null,
+      "intPartitionOptions": null
+    },
+    {
+      "name": "biosample",
+      "columns": [
+        {"name": "biosample_id", "datatype": "string", "array_of": false, "required": true},
+        {"name": "anatomical_site", "datatype": "string", "array_of": false},
+        {"name": "apriori_cell_type", "datatype": "string", "array_of": true},
+        {"name": "biosample_type", "datatype": "string", "array_of": false},
+        {"name": "diagnosis_id", "datatype": "string", "array_of": true},
+        {"name": "disease", "datatype": "string", "array_of": false},
```

Comment:
I would think a biosample could be associated with more than one disease, could it not?
Wondering if this should be an array.

Reply:
I think this is a really good question. I've added to https://broadworkbench.atlassian.net/browse/DI-121 (where I'm consolidating potential FSS schema updates).